client/src/layoutRoutes/HeaderBottomNavLayoutRoute.js

```diff
@@ -25,7 +29,22 @@ const HeaderBottomNavLayoutRoute = ({ component: Component, ...rest }) => {
           <MainContainer>
             <Component {...matchProps} />
           </MainContainer>
-          <BottomNav />
+          <BottomNav
+            links={[
```

Comment:
very cool approach btw :)

Reply:
gotta admit I copied this from the others who worked up this approach with Leon 😉 so I cheated.. that's why it resulted also in the other stuff you commented 🙈